### DISCUSSION TITLE
MLM2PRO Bluetooth Range

### DISCUSSION THREAD

**Discussion by GatorSwamp** [springbok/MLM2PRO-GSPro-Connector#104] [Q&A]
Hi there!

I've been using the OCR version of this app for 6 months or so. It's dramatically improved since it launched, so thank you!

I have my PC about 40ft away from the launch monitor when I try to directly connect via bluetooth. It seems like the launch monitor will only connect, and stay connected if it stays around 20ft. Does this sound right? Speakers and headphones work flawlessly in the same location. Would take some major reconfiguring of my setup to get my PC closer to the launch monitor. Wondering if there's a way to extend the bluetooth range. OCR version is fine. But the direct bluetooth would be preferable.

Thanks!

**Comment by wonder99** [comment]
I purchased a powered USB extender on Amazon and plugged in a USB Bluetooth antenna to that. My antenna is now about 1 ft from the MLM and I get about -65 or -70 dBm signal strength. Unfortunately some MLMs have very poor antennas and the distance must be very very short

**Reply by GatorSwamp** [reply]
My phone has to be pretty close to the launch monitor to start the initial connection when it's in BT mode. So I bet you're right about the antenna. I'm also pretty sure I have one of the initial waves of production. 

I have a USB-BT antenna. So I'll look into getting the USB-powered extension - Thanks!
